pinot-core/src/main/java/org/apache/pinot/core/data/manager/offline/DimensionTableDataManager.java

```diff
@@ -116,7 +116,7 @@ public void removeSegment(String segmentName) {
       _logger.info("Successfully removed segment {} and reloaded lookup table: {}", segmentName, getTableName());
     } catch (Exception e) {
       throw new RuntimeException(String
-          .format("Error reloading lookup table after segment remove ({}) for table: {}", segmentName, getTableName()),
+          .format("Error reloading lookup table after segment remove (%s) for table: %s", segmentName, getTableName()),
```

Comment:
Thanks for finding and fixing this.

Minor nit: Fix grammar, and use single quotes around %s (I usually do that to catch an empty string value as '').

Something like:
"Error reloading lookup table after segment '%s' removed for table: '%s'"

Reply:
Emm... What's the difference?

Reply:
Anyway, I will address your comment. :sweat_smile: